src/main/java/uk/gov/hmcts/ccd/domain/model/common/CatalogueResponseCode.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+public enum CatalogueResponseCode {
+
+    // CCD.00 Success codes
+
+    // CCD.01 Validation codes
+    VALIDATION_INVALID_DATA(CatalogueResponseGroup.VALIDATION, "01", "Invalid data provided."),
+
+    // CCD.02 Callback codes
+    CALLBACK_FAILURE(CatalogueResponseGroup.CALLBACK, "01", "Callback failure."),
+    CALLBACK_PAYMENT_REQUIRED(CatalogueResponseGroup.CALLBACK, "02", "Payment required by callback.");
+
+    final CatalogueResponseGroup group;
+    final String code;
+    final String message;
+
+    CatalogueResponseCode(final CatalogueResponseGroup group, final String code, final String message) {
```

Comment:
The word 'code' is used in multiple places with different content/meaning. Better to rename. I suggest to use an integer for this, and name it as "sequenceNo". The getCode() should then return a formatted String to contain ".01" for sequence is 1, for example. There should be a method "int getSequenceNo()", and no attribute like 'code' in the class. An attribute X and a method getX containing/returning different values is not good.